tomviz/python/Recon_live_tomo.py

```diff
@@ -0,0 +1,69 @@
+import tomviz.operators
+import numpy as np
+import paramiko
+import h5py
+
+
+class liveTomoOperator(tomviz.operators.CancelableOperator):
+
+    def transform(self, dataset, hostName=None, userName=None, password=None,
+                  port=None, filename=None, remoteDirectory=None,
+                  localDirectory=None):
+
+        transport = paramiko.Transport((hostName, port))
+        transport.connect(username=userName, password=password)
+        sftpConnection = paramiko.SFTPClient.from_transport(transport)
+        sftpConnection.chdir(remoteDirectory)
+
+        # Create child for recon
+        child = dataset.create_child_dataset()
+        lastMtime = 0
+        firstLoad = True
+        while True:
+
+            if self.canceled:
+                break
+
+            # Check is new volume is available
+            (newVolume, lastMtime) = check_for_new_volume(sftpConnection, 
+                                      filename, localDirectory, lastMtime)
+            if newVolume:
+                try:
+                    file = h5py.File(localDirectory + filename, 'r')
+                    if firstLoad:
+                        recon = np.zeros(file['recon'].shape, dtype=np.float32)
+                        firstLoad = False
+                    recon[:, :, :] = file['recon'][:, :, :]
+                    child.active_scalars = recon
+                    self.progress.data = child
+                    file.close()
+                except: 
+                    continue
```

Comment:
Are you sure you want to ignore the potential exceptions here? If there is a specific exception class that you want to ignore I would add in the type in the except block. So called blind except blocks are generally a bad idea.